tests/grib_util_set_spec2.cc

```diff
@@ -21,13 +21,9 @@ static void dump_it(grib_handle* h)
 static grib_handle* test0()
 {
     fprintf(stderr, "Doing test %s\n-----------------\n", __func__);
-    int err                  = 0;
-    grib_util_grid_spec spec = {
-        0,
-    };
-    grib_util_packing_spec packing_spec = {
-        0,
-    };
+    int err = 0;
+    grib_util_grid_spec spec = {0,};
+    grib_util_packing_spec packing_spec = {0,};
```

Comment:
The changes of this sort were made by clang-format (I'm only trying to satisfy the convention)

Reply:
Yes. I know ;)